offline/kvm-hetzner.md

```diff
@@ -80,11 +85,12 @@ tar -xzf ../wire-server-deploy-static-*.tgz
 tar -xf debs.tar
 ```
 
-### Install Docker from debian archive.
+### (FIXME: add iptables) Install Docker from debian archive.
```

Comment:
I think you can drop the FIXME since you did add the `apt install iptables` line below.